pkg/grpc/gen/remote.go

```diff
@@ -20,10 +20,13 @@ import (
 	"github.com/talos-systems/talos/pkg/machinery/constants"
 )
 
-var trustdResolverScheme string
+var (
+	trustdResolverScheme string
+	resolverDefaultPort  int = constants.TrustdPort
+)
 
 func init() {
-	trustdResolverScheme = resolver.RegisterRoundRobinResolver(constants.TrustdPort)
+	trustdResolverScheme = resolver.RegisterRoundRobinResolver(resolverDefaultPort)
```

Comment:
why do we need to pass port the resolver? if we append the port outside the resolver??